chart/kubedb-catalog/templates/mongodb.yaml

```diff
@@ -209,6 +209,7 @@ metadata:
   labels:
         {{- include "kubedb-catalog.labels" . | nindent 4 }}
 spec:
+  deprecated: true
```

Comment:
Do we need to deprecate v3? or just note that new ssl feature is supported by v4 image?